common/src/main/java/com/wynntils/features/inventory/HorseMountFeature.java

```diff
@@ -36,7 +36,7 @@
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 import org.lwjgl.glfw.GLFW;
 
-@ConfigCategory(Category.COMBAT)
+@ConfigCategory(Category.INVENTORY)
 public class HorseMountFeature extends Feature {
```

Comment:
I disagree. This is more akin to QuickSpellCast. The fact that you need to have an horse in the inventory is not really relevant, just as it is "irrelevant" that you need to hold a weapon for QuickSpellCast to work. That does not mean it is "Inventory".

I agree that "Combat" sounds misleading, though. What if rename Combat to "Gameplay"?